docs/linux.md

```diff
@@ -24,7 +24,7 @@ sudo yum -y install gcc gcc-c++ python3-setuptools git-clang-format dos2unix rub
 ```
 ## 2. Packages Requirements when Executing Binaries:
 
-OpenSSL* toolkit version 1.1.1s
+OpenSSL* toolkit version 3.0.7
 GCC version > 7.5
 Curl version 7.86
 
```

Comment:
In case if opensssl is already manually installed - same version may need to be uninstalled using script . Can we capture this point here ?